README.md

```diff
@@ -94,33 +103,49 @@ For some Linux distributives internal packages with `sqlite_fdw` are avalilable.
 ### Source installation
 
 Prerequisites:
-* `libsqlite3-dev`, especially `sqlite.h`
-* `postgresql-server-dev`, especially `postgres.h`
 * `gcc`
 * `make`
+* `postgresql-server-dev`, especially `postgres.h`
+* `libsqlite3-dev`, especially `sqlite.h`
+* `libspatialite-dev` only for geoinformational data types support (SpatiaLite ↔ PostGIS) or full tests
 
 #### 1. Install SQLite & Postgres Development Libraries
 
 For Debian or Ubuntu:
+
 `apt-get install libsqlite3-dev`
+
 `apt-get install postgresql-server-dev-XX`, where XX matches your postgres version, i.e. `apt-get install postgresql-server-dev-15`
 
-You can also [download SQLite source code][1] and [build SQLite][2] with FTS5 for full-text search.
+`apt-get install libspatialite-dev` - for SpatiaLite ↔ PostGIS transformations
+
+Instead of `libsqlite3-dev` you can also [download SQLite source code][1] and [build SQLite][2] with FTS5 for full-text search.
 
 #### 2. Build and install sqlite_fdw
 
-Add a directory of `pg_config` to PATH and build and install `sqlite_fdw`.
+You can compile without `libspatialite-dev` and GIS support. This library is necessary only for full tests. PostGIS is not necessary for `sqlite_fdw` compilation anyway and also used only for full tests.
 
+Before building please add a directory of `pg_config` to PATH or ensure `pg_config` program is accessable from command line only by the name.
```

Comment:
This means shell can execute `pg_config`, not only `/usr/bin/pg_config` or `/usr .... postgresql17/bin/pg_config`. This is common `make` requirement in case of `USE_PGXS=1`.

Reply:
Thank you for your explanation. Please help me to resolve this comment.